sulong/projects/com.oracle.truffle.llvm.test/src/com/oracle/truffle/llvm/test/interop/MutexTypeCheckTest.java

```diff
@@ -47,7 +47,7 @@
 import org.junit.runners.Parameterized.Parameters;
 
 @RunWith(Parameterized.class)
-public final class TypeCheckTest extends InteropTestBase {
+public final class MutexTypeCheckTest extends InteropTestBase {
```

Comment:
why has this been renamed?

Reply:
also by accident, renamed back to original now.